source/reporters/ut_realtime_reporter.tpb

```diff
@@ -57,25 +57,27 @@ create or replace type body ut_realtime_reporter is
       self.print_node('name', a_suite.name);
       self.print_node('description', a_suite.description);
       <<suite_elements>>
+      self.print_start_node('items');
       for i in 1 .. a_suite.items.count loop
         if a_suite.items(i) is of(ut_test) then
           print_test_elements(treat(a_suite.items(i) as ut_test));
         elsif a_suite.items(i) is of(ut_logical_suite) then
           print_suite_elements(treat(a_suite.items(i) as ut_logical_suite));
         end if;
       end loop suite_elements;
+      self.print_end_node('items');
       self.print_end_node('suite');
     end print_suite_elements;
   begin
     xml_header := ut_utils.get_xml_header(a_run.client_character_set);
     self.print_xml_fragment(xml_header);
     self.print_start_node('event', 'type', 'pre-run');
-    self.print_start_node('suites');
+    self.print_start_node('items');
     <<items>>
     for i in 1 .. a_run.items.count loop
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Replace this FOR loop by a more robust WHILE one. ([plsql:ForLoopUsageCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AForLoopUsageCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=819&issues=AWhDYNbDXONnvcWDUk4T&open=AWhDYNbDXONnvcWDUk4T)

[ISSUE_KEY]: AWhDYNbDXONnvcWDUk4T